README.md

```diff
@@ -0,0 +1,25 @@
+# EJB test suite
+
+This is upstream EJB integration TS.
+
+# Quick start
+
+The following commands starts the whole TS with latest upstream bits:
+
+```bash
+mvn -B -ntp package -DspecificModule=prepare
+mvn -B -ntp dependency:tree clean verify --fail-at-end
+```
+
+# TS structure
+
+There are three modules in this TS, see the details in following links:
+
+* [basic client tests](basic)
+* [multinode tests](multinode)
+* [timers tests](timers)
+
+[Basic client tests](basic) and [multinode tests](multinode) can be started with one those options:
+
+* Use bom files. With this configuration, the documentation above describes `-Dserver.home` or `server.zip` parameters. As an alternative for using one of those properties, you can also build server with the latest upstream bits by `mvn -B -ntp package -DspecificModule=prepare` command and then not use `-Dserver.home` or `server.zip` parameters.
```

Comment:
```suggestion
* Use BOMs to govern client dependencies. With this configuration, the documentation above describes `-Dserver.home` or `-Dserver.zip` parameters. As an alternative for using one of those properties, you can also build server with the latest upstream bits by `mvn -B -ntp package -DspecificModule=prepare` command and then not use `-Dserver.home` or `-Dserver.zip` parameters.
```

Reply:
also what "documentation above" refers to? I don't see description of those parameters anywhere in this file.

Reply:
Updated, thanks!

> also what "documentation above" refers to? I don't see description of those parameters anywhere in this file.

This file contains links to basic and multinode subfolders with readme where those parameters are described: https://github.com/marekkopecky/ejb-client-testsuite/blob/readme/README.md